fairscale/utils/testing.py

```diff
@@ -144,8 +152,7 @@ def spawn_for_all_world_sizes(test_func: Callable, world_sizes: List[int] = get_
 
     for world_size in world_sizes:
         filename = tempfile.mkstemp()[1]
-        context = mp.spawn(test_func, args=(world_size, filename, *args), nprocs=world_size, join=False)  # type: ignore
-        context.join(timeout=60.0)
+        mp.spawn(test_func, args=(world_size, filename, *args), nprocs=world_size, join=True)  # type: ignore
```

Comment:
turns out join=False was a bad idea (not sure why), the intent was to get a stack when the process hangs and pytest-timeout is a better solution for that anyway

Reply:
add a comment about this?